systemd/hera-hashpipe-monitor.service

```diff
@@ -0,0 +1,20 @@
+# Configuration file for systemd that keeps the HERA hashpipe monitor running.
```

Comment:
Historically we've put these services in the commissioning repo. In a way it makes more sense here, but people might not find it here if they expect to find it in that repo.

Reply:
I've gone back and forth on this myself. There is one other systemd files here which we could move to commissioning repo, but maybe we would want to re-organize the directory where the system files are held to show what runs where? 

I'm not sure what the best plan would be.

Reply:
Yeah, I think this needs to be a wider discussion with @plaplant @david-deboer @dannyjacobs and maybe others

Reply:
we decided on the Commissioning call to leave it here.